UI and API tests for conduit application

vanitha128 opened the pull request:
This pull request contains two sets of tests
1) UI and API tests using Java, selenium, RestAssured.
2) UI tests using Javascript and Cypress.

More information regarding the tests and run results are available in readMe file

vanitha128 committed:
First cut of conduit front end and api tests

vanitha128 committed:
formatting

vanitha128 committed:
cypress tests for conduit UI testing

vanitha128 committed:
screenshots of test results

vanitha128 committed:
Update README.md

vanitha128 committed:
api test results

vanitha128 committed:
Update README.md

vanitha128 committed:
deleting unwanted files

vanitha128 committed:
Update README.md

linzcrowe commented:
Thanks @vanitha128 . Can you please add 2 things for me?

The first is a some documentation or tooling to help setup and run the tests. package.json script entries, shell scripts, documentation in the readme, whatever will help take a consumer from cloning the repository to successfully running the tests.

The second is your suggestions for quality improvements, as that section of the readme is currently blank. Since we're chatting today that could just be bullet points and we chat through detail at 3pm.

vanitha128 committed:
enhancements

vanitha128 commented:
@linzcrowe Thanks. I have updated the necessary scripts to run the test files for both cypress and selenium tests. I have also updated the quality improvements section.

vanitha128 committed:
Docker files for cypress(done) and selenium(in progress)

vanitha128 committed:
script files for docker build and run
